features/wijzigingen.feature

```diff
@@ -0,0 +1,100 @@
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" een item met de waarde "999992855"
+
+    Scenario: Gebruiker zet een volgindicatie met een einddatum in het verleden of vandaag
+        Gegeven gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999992806" met einddatum "2021-03-15"
+        En op 12 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999992806"
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" GEEN item met de waarde "999992806"
+    
+    Scenario: Gebruiker heropent de volgindicatie
+        Gegeven gebruiker "A" op 17 januari 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999994220" met einddatum "2021-03-01"
+        En gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999994220" en geen einddatum
+        En op 12 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999994220"
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" een item met de waarde "999994220"
+
+    Scenario: Gebruiker heropent de volgindicatie en wijzigingen vóór het heropenen worden niet gemeld
+        Gegeven gebruiker "A" op 17 januari 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999992351" met einddatum "2021-03-01"
+        En op 21 februari 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999992351"
+        En gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999992351" en geen einddatum
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen
+        Dan bevat het antwoord in "burgerservicenummers" GEEN item met de waarde "999992351"
+
+    Scenario: Gebruiker vraagt expliciet wijzigingen van vóór het zetten van de volgindicatie
+        Gegeven op 4 maart 2021 is in de BRP registratie een wijziging doorgevoerd in de gegevens van de persoon "999992971"
+        En gebruiker "A" op 5 maart 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "999992971" en geen einddatum
+        Als gebruiker "A" op 15 maart 2021 de wijzigingen vraagt met /wijzigingen?vanaf=2021-03-01
+        Dan bevat het antwoord in "burgerservicenummers" GEEN item met de waarde "999992971"
+
+    Scenario: Gebruiker verlengt de volgindicatie en eerdere wijzigingen worden wel gemeld
+        Gegeven gebruiker "A" op 17 januari 2021 een volgindicatie heeft gezet op de persoon met burgerservicenummer "000009945" met einddatum "2021-03-05"
```

Comment:
Dit is samen met scenario `Gebruiker heropent de volgindicatie en wijzigingen vóór het heropenen worden niet gemeld` heel raar gedrag. In die scenario krijgt de gebruiker een wijziging van 21 februari niet door omdat hij zijn volgindicatie heeft laten verlopen en dan pas 'verlengt'. In deze scenario krijgt de gebruiker de wijziging wel omdat hij zijn volgindicatie verlengt voordat deze verloopt.

Reply:
Dat lijkt denk ik maar zo, omdat er weinig tijd tussen zit. Ik denk daarbij aan bijvoorbeeld een uitkeringensysteem. Een persoon vraagt in juni 2018 een uitkering aan, en die eindigt in januari 2019 (bijv. omdat de persoon werk heeft gevonden). Dan eindigt het volgen van de persoon ook in januari 2019. In april 2021 verliest deze persoon weer zijn werk en vraagt opnieuw de uitkering aan. Dus wordt de persoon dan vanaf april 2021 weer gevolgd. Als dit uitkeringensysteem dan op 9 april /wijzigingen doet, moet deze persoon (ben) niet worden geleverd om een wijziging die bijv. in 2020 is geregistreerd. 
Vooral voor een buitengemeentelijke persoon zou dat ook niet kunnen, omdat de persoon in de tussentijd niet is gevolgd. Het is dan toeval of wijzigingen uit de tussentijd kunnen worden gemeld, omdat dezelfde persoon door een andere gebruiker wordt gevolgd of omdat de persoon binnengemeentelijke is/was.